src/TimeSpans.jl

```diff
@@ -296,9 +291,8 @@ TimeSpan(3000000000 nanoseconds, 6000000000 nanoseconds)
 """
 function time_from_index(sample_rate, sample_range::AbstractUnitRange)
     i, j = first(sample_range), last(sample_range)
-    j = j == i ? j : j + 1
     return TimeSpan(time_from_index(sample_rate, i),
-                    time_from_index(sample_rate, j))
+                    time_from_index(sample_rate, j + 1))
```

Comment:
This matches what I did in AlignedSpans:
https://github.com/beacon-biosignals/AlignedSpans.jl/blob/21d2e331f0d861ef6bed35236b550fb688a23459/src/interop.jl#L45